src/modules/localization/src/wheel_odom/odometry.cpp

```diff
@@ -129,53 +127,55 @@ namespace wheel_odometry_node
         ///////////////////////
         
         // velocity estimate in x and y
-        x_velocity = vel_magnitude * cos(yaw_estimate); // [m/s]
-        y_velocity = vel_magnitude * sin(yaw_estimate); // [m/s]
-
+        // x_velocity = vel_magnitude * cos(yaw_estimate); // [m/s]
+        // y_velocity = vel_magnitude * sin(yaw_estimate); // [m/s]
+        x_velocity = vel_magnitude;
+        // y_velocity = 0;
         // twist messages
         velocity_estimate.twist.linear.x = x_velocity; // [m/s]
-        velocity_estimate.twist.linear.y = y_velocity; // [m/s]
+        // velocity_estimate.twist.linear.y = y_velocity; // [m/s]

```

Comment:
When would we use `y_velocity` in the future?

Reply:
It's unlikely, I can delete the excess within `odometry.cpp`